tests/core/gui/test_workspaces_reencrypt.py

```diff
@@ -121,17 +133,18 @@ def _reencrypt_button_displayed():
 
 @pytest.mark.gui
 @pytest.mark.trio
+@pytest.mark.flaky(reruns=3)
```

Comment:
what's that ? the PR was talking about typing, not flaky tests :/

Reply:
In fact those test are flaky but don't marked as so.
(They don't have the fix you've made)

Reply:
I've removed them